azure-mgmt-sql/src/test/java/com/microsoft/azure/management/sql/SqlServerOperationsTests.java

```diff
@@ -188,13 +191,13 @@ public void canManageReplicationLinks() throws Exception {
                 .withCreateMode(CreateMode.ONLINE_SECONDARY)
                 .create();
         Thread.sleep(2000);
-        PagedList<ReplicationLink> replicationLinksInDb1 = databaseInServer1.replicationLinks().list();
+        List<ReplicationLink> replicationLinksInDb1 = databaseInServer1.replicationLinks().list();
```

Comment:
List here is just collection of the objects, so Set will also work here. Consider it similar to ListByResourceGroup or List operations. I see that SupportsListing returns PagedList, although I have almost all the code ready returning List rather than PageList in SQL. But we have to agree on one of the type. In C# I would have suggested IEnumerable for this, in Java Iterable<T> is equivalent to this.